src/main/java/org/itsallcode/openfasttrace/importer/ImporterContext.java

```diff
@@ -0,0 +1,52 @@
+package org.itsallcode.openfasttrace.importer;
+
+/*-
+ * #%L
+ * OpenFastTrace
+ * %%
+ * Copyright (C) 2016 - 2018 itsallcode.org
+ * %%
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ * 
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public
+ * License along with this program.  If not, see
+ * <http://www.gnu.org/licenses/gpl-3.0.html>.
+ * #L%
+ */
+import java.util.Objects;
+
+import org.itsallcode.openfasttrace.importer.legacytag.config.LegacyTagImporterConfig;
+
+public class ImporterContext
+{
+    private final LegacyTagImporterConfig tagImporterConfig;

```

Comment:
This looks like unintended information disclosure to me.
If all importers get this context, all have access to the `tagImporterConfig` - even if they are not the `TagImporter`.

There must be a better way where only the `TagImporter` learns about its config.

Reply:
I agree. I propose to address this in #144